tst/testinstall/grppc.tst

```diff
@@ -52,6 +139,11 @@ gap> List(sys,i->Length(AsList(i)));
 gap> G := SmallGroup( 144, 183 );;
 gap> F := FittingSubgroup( G );;
 gap> S := SylowSubgroup( F, 2 );;
+gap> Display(Image(IsomorphismPcGroup(S)));
+pc-group with relations:
+g1^2 = id
+g2^2 = id
+All other pairs of generators commute.
```

Comment:
Note that no pairs of generators were printed, making this message in this particular situation somewhat confusion (even though it is of course formally correct). Perhaps one could refine the message a bit, by doing something like  `if IsAbelian(grp) then Print("all generators commute, the groups is abelian\n") else ...` or so. 

Of course this is super nitpicky. I am also OK with it if it stays as it is.